.eleventy.js

```diff
@@ -2,14 +2,23 @@ const {EleventyServerlessBundlerPlugin} = require('@11ty/eleventy');
 const serialize = require('serialize-javascript');
 const {titleCase} = require('title-case');
 const {isWebUri} = require('valid-url');
-
+const markdownIt = require('markdown-it');
+const markdownItFootnote = require('markdown-it-footnote');
 /**
  * @typedef {import('@11ty/eleventy/src/UserConfig')} EleventyConfig
  * @typedef {ReturnType<import('@11ty/eleventy/src/defaultConfig')>} EleventyReturnValue
  *
  * @type {(eleventyConfig: EleventyConfig)) => EleventyReturnValue}
  */
 module.exports = function (eleventyConfig) {
+	let markdownItOptions = {
+		html: true,
+		breaks: true,
+		linkify: true,
+	};
+	let markdownLib = markdownIt(markdownItOptions).use(markdownItFootnote);
```

Comment:
Because it's common to use multiple Markdown plugins, I write this like:

```suggestion
	let markdownLib = markdownIt({
		html: true,
		breaks: true,
		linkify: true,
	});
	markdownLib.use(markdownItFootnote);
```

This way, it's a little nicer to `.use()` for additional plugins.

Reply:
Actually, I'm wrong, lol